frontend/src/ol/dataLayers/layerTypes/points.js

```diff
@@ -7,7 +7,7 @@ import boxReloadVectorSourceLoader from '../boxReloadVectorSourceLoader';
 
 const pointsLayer = (layerData, handleIsLoading) => {
   let vectorSource;
-  if (layerData.geoReferenceId === null) {
+  if (layerData.geoReferenceId === null || (layerData.layerOptions && layerData.layerOptions.boxReload)) {
```

Comment:
I guess every layer now have geoReferenceId so we could get rid of layerData.geoReferenceId === null?

Reply:
Let's keep it there, there might be old data, or another api :)